Resources/Assets.xcassets/weather/averses.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I think we should use English for asset names, you can get weather types in GraphiQL
(same for all weather assets)